src/zino/events.py

```diff
@@ -144,6 +147,21 @@ def commit(self, event: Event, user: str = "monitor"):
 
         self._call_observers_for(new_event=event, old_event=old_event)
 
+    def delete(self, event: Event):
+        """Removes an event from the events dict and notifies all observers"""
+        try:
+            del self.events[event.id]
+        except KeyError:
+            pass
+        self._call_observers_for(new_event=event)
+
+    def delete_closed_events(self):
+        """Deletes all events that have been closed for at least 8 hours"""
```

Comment:
No need to mention the number 8 hours in the docstring here.  It's defined by a constant.  Now, if you change the constant, you also have to remember to change the docstring...